### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by theBwoyProgrammer**
Added mobile menu and redirection to particular section when click on the nav-links

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -34,14 +34,62 @@ main {
...
+  flex-direction: column;
+  text-align: start;
+  transition: 1s;
+  width: 100%;
+  justify-content: center;
+  margin-right: 200px;
+  z-index: 200;
+  background-color: #fff;
+  height: 100%;
+}
```
Comment (vechicin) [inline styles.css]: I cannot see the `X` close button.

Screen Shot 2022-06-20 at 9 51 27

Please add it.

Additionally, your design doesn't follow the template design.

Screen Shot 2022-06-20 at 9 54 42

Please fix it.

Screen Shot 2022-06-20 at 9 56 22

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @HatimDev-HE ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good first attempt at implementing the mobile menu

## Required Changes ♻️

- [ ] You need to set up ESLint, the linters for JavaScript, please do so.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/HatimDev-HE/Portfolio-site-4/blob/61c6390e881fc9ccbdb41ce860c795fe7159b036/styles.css#L144-L154

I'm not sure what happened, but your desktop design is completely thrown out of whack. I am using Google Chrome, for reference.

Screen Shot 2022-06-20 at 10 01 09

Please fix it.

File: styles.css [hunk 2]
```diff
@@ -34,14 +34,64 @@ main {
...
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+  margin-right: -33px;
+}
```
Comment (leonardodiasb) [inline styles.css]: - [ ] Your "X" button is partially visible. Looks like the menu-mobile is overlapping it. You could kindly set a higher z-index for the `.hamburger` class over the menu mobile z-index.
- [OPTIONAL] Your hamburger button is almost being cut out of the screen. Kindly review the margin-right of that button.
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @HatimDev-HE , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @HatimDev-HE ,  

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- menu works well
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**